R/build_lm.R

```diff
@@ -539,7 +557,8 @@ build_lm.fast <- function(df,
       else {
         class(rf) <- c("lm_exploratory", class(rf))
       }
-      rf
+      list(rf = rf, test_index = test_index, source_data = source_data)
```

Comment:
Not sure why this would work, adding extra layer of list above rf. Let's discuss at meeting.

Reply:
The elements  of this list is later flattened into separate columns.